openfisca_core/tracers.py

```diff
@@ -224,14 +224,14 @@ def print_trace(self, variable_name, period, extra_params = None, max_depth = 1,
         def _print_details(key, depth):
             if depth > 0 and ignore_zero and np.all(self.trace[key]['value'] == 0):
                 return
-            self._print_node(key, depth, aggregate)
+            yield self._print_node(key, depth, aggregate)
             if depth < max_depth:
                 for dependency in self.trace[key]['dependencies']:
-                    _print_details(dependency, depth + 1)
+                    return _print_details(dependency, depth + 1)
 
-        _print_details(key, 0)
+        return _print_details(key, 0)
 
-    def print_computation_log(self, aggregate = False):
+    def computation_log(self, aggregate = False):
         """
```

Comment:
@Morendil  this doc string should be moved to the `print_computation_log` method (this causes a warning in the doc build)